src/AppConfig.ts

```diff
@@ -6,6 +6,10 @@ export const appConfig = {
   JWT_SECRET: get('appConfig.jwtSecret'),
   ACCESS_TOKEN_EXPIRE_MIN: get('appConfig.accessTokenExpireMin'),
   REFRESH_TOKEN_EXPIRE_MIN: get('appConfig.refreshTokenExpireMin'),
+  EMAIL_TOKEN_EXPIRE_DAY: get('appConfig.emailTokenExpireDay'),
+  RESET_TOKEN_EXPIRE_DAY: get('appConfig.resetTokenExpireDay'),
```

Comment:
You can set days or minutes in config file, e.g. like: resetTokenExpireTime: '1d', and : accessTokenExpireTime: '30m'.